library/oidc.c

```diff
@@ -417,6 +427,30 @@ static void ext_done(uv_work_t *wr, int status) {
 }
 
 static void start_ext_auth(auth_req *req, const char *ep, int qc, tlsuv_http_pair q[]) {
+    uv_loop_t *loop = req->clt->timer->loop;
+    struct sockaddr_in6 addr = {
```

Comment:
what happens when the OS doesn't support ipv6 or has it disabled? lots of the internet was recommending disabling ipv6 for a long time, i'm just wondering if this will be hard to understand/debug and if we can give the user a decent error